### ISSUE TITLE
niivue multiplanarForceRender option is getting a name change

### ISSUE THREAD

**Issue by hanayik** [afni/afni#669]
@mrneont , we are considering making a breaking change to the `multiplanarForceRender` option in NiiVue. 

I can see that this is used in your javascript [here](https://github.com/afni/afni/blob/f8853eccdc8dacb9e1bfa289adfe0093045e7d3c/src/python_scripts/afnipy/lib_apqc_niivue.py#L286). 

The discussion on this topic is in [this](https://github.com/niivue/niivue/issues/1024) NiiVue issue. 

Ultimately, the changes are aimed at making this option more interpretable and allows developers to control the behaviour of the 3D rendering tile when using the multiplanar layout in NiiVue. 

The proposed changes will:
- change `multiplanarForceRender` --> `multiplanarShowRender`
- change type to `string` with allowed values being: `"always" | "never" | "auto"`
- allow completely hiding the 3D render tile in multiplanar mode (which was previously not possible)

I'm happy to open a PR with the proper changes in the AFNI code once this change is merged into the NiiVue main branch.

**Comment by mrneont** [comment]
@hanayik : 

Hmm, OK.  Would it be possible to let `multiplanarForceRender = true` still exist, mapping it to `multiplanarShowRender = "always"`, I guess?

Or, would the default behavior of `multiplanarShowRender` be `"always"`, anyways, so that not having it specificed would yield the same behavior as `multiplanarForceRender = true`?  That would be less breaking, as long as calling `multiplanarForceRender` does not produce an error.

--pt

**Comment by hanayik** [comment]
@mrneont , this work has been published in Niivue now. It has been done in a way similar to your suggestion. That means there are no modifications to AFNI code required. However, to have the full control over the render tile in multiplanar view you would need to switch to the `multiplanarShowRender` option.  

[Here](https://github.com/niivue/niivue/blob/c0a701cc11ed554f17c165064c4d76d30dc33a6b/demos/features/afni.html#L58) is an example use of the new enum `SHOW_RENDER`. or you can supply the `multiplanarShowRender` option when initialising a niivue instance via the initial options object you pass in. 

Again, there is no change need in the AFNI code to maintain your current functionality.

**Comment by mrneont** [comment]
@hanayik: Just reopening to say thanks, and I'm glad that was workable in this way.

--pt
